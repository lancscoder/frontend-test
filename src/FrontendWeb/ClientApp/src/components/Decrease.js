import React from 'react';
import Glyph from './Glyph';

const Decrease = () => (
  <Glyph name="glyphicon glyphicon-arrow-down icon-decrease" />
);

export default Decrease;