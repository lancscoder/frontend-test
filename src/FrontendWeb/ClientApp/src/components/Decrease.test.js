import React from 'react';
import renderer from 'react-test-renderer';
import Decrease from './Decrease';

describe('Decrease ::', () => {
    it('should render with correct class', () => {
        const component = renderer.create(<Decrease />);;

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();

        expect(tree.props.className).toBe('glyphicon glyphicon-arrow-down icon-decrease');
    });
});