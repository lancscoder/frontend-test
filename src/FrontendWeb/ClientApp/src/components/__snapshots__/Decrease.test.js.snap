// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Decrease :: should render with correct class 1`] = `
<span
  aria-hidden="true"
  className="glyphicon glyphicon-arrow-down icon-decrease"
  style={
    Object {
      "paddingLeft": "5px",
      "paddingRight": "5px",
    }
  }
/>
`;
